target/linux/realtek/base-files/lib/upgrade/platform.sh

```diff
@@ -4,6 +4,21 @@ REQUIRE_IMAGE_METADATA=1
 RAMFS_COPY_BIN='fw_printenv fw_setenv'
 RAMFS_COPY_DATA='/etc/fw_env.config /var/lock/fw_printenv.lock'
 
+# The TP-Link bootloader gets its flash layout from "bootargs".
+# Use this to our advantage, and:
+#  1. Disable the second rootfs (usrimg2)
+#  2. Extend the first rootfs (usrimg1) to include more space
+#  3. Increase the baudrate to the expected 115200n8
```

Comment:
Why is the expected baud rate 115200? If the devices use something else, then you could override `/chosen/bootargs` in DT instead.

Yes, I know who authored commit af2cfbda2bf544666b30b2730e9486b2d6eff76f to consolidate the bootargs in the SoC DTSI, and realise I have argued otherwise about "not just using defaults because all devices we know do it that way". (In my defence, it looks like I was mostly replacing a default that was never actually used.)

Reply:
115200 is the de-facto standard.

The bootloader is painfully slow as it is. Slowing the boot again with a slow console is infuriating. I saw an opportunity to improve things, and I took it.